packages/libs/pactjs/src/IPactCommand.ts

```diff
@@ -0,0 +1,38 @@
+/**
```

Comment:
We have `ICap`, `IMetaData (=IPublicMeta)`, `ISignedTransaction(= ISignedCommand)` , `ICommandPayload(=IUnsignedTransactionCommand)` , etc in @kadena/types : https://github.com/kadena-community/kadena.js/blob/master/packages/libs/types/src/PactCommand.ts
Should we have one place for all pact value / tx types?

Reply:
This is great! I'll use those from @kadena/types. I'm still not sure what would be the best way to define types. I'm much a proponent of defining the type at the place where it's being used, not separate them too much... So I'm conflicted about how we should use this.